MakeMKV Missing Dependancies post 1.17.6

Linked issue from ARM [ 🐛 MakeMKV Bug - Bluray ripping not functioning in ARM #1195](https://github.com/automatic-ripping-machine/automatic-ripping-machine/issues/1195)

MakeMKV have released a new version of software, 1.17.8
Quick testing, seems that it requires a newer version of libc

ARM running [phusion/baseimage:focal-1.2.0](https://hub.docker.com/layers/phusion/baseimage/focal-1.2.0/images/sha256-c718615357c28fa21c71d0e7692637d8b9455ee8925b0c89163f5685a1ca8133?context=explore)

```bash
# makemkvcon -v
makemkvcon: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by makemkvcon)
makemkvcon: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by makemkvcon)

# uname -r
5.15.0-122-generic

# lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
```

currently libc6 can't be updated

```bash
# apt install libc6
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libc6 is already the newest version (2.31-0ubuntu9.16).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
```

Testing proposed solution in docker container, as suggested in ARM issue [Stackoverflow](https://stackoverflow.com/questions/78711302/lib-x86-64-linux-gnu-libc-so-6-version-glibc-2-33-not-found)

```bash
echo "deb http://security.ubuntu.com/ubuntu jammy-security main" >> /etc/apt/sources.list
apt update
apt install libc6
```

makemkvcon returns:

```bash
# makemkvcon -v
makemkvcon: invalid option -- 'v'
Use: makemkvcon [switches] Command [Parameters]

Commands:
  info <source>
      prints info about disc
  mkv <source> <title id> <destination folder>
      saves a single title to mkv file
  backup <source> <destination folder>
      backs up disc to a hard drive
  f <args>
      run universal firmware tool
  reg <key string or file name>
      enter registration key into program

Source specification:
  iso:<FileName>    - open iso image <FileName>
  file:<FolderName> - open files in folder <FolderName>
  disc:<DiscId>     - open disc with id <DiscId> (see list Command)
  dev:<DeviceName>  - open disc with OS device name <DeviceName>

Switches:
  -r --robot        - turn on "robot" mode, see http://www.makemkv.com/developers
```

However, running the [install_makemkv.sh](https://github.com/automatic-ripping-machine/arm-dependencies/blob/main/scripts/install_makemkv.sh) script fails, reporting a gcc error.

Rebuilding the container and running again, confirmed.
1. Running install_makemkv.sh works, no issues reported
2. Makemkv reports an error `GLIBC_2.33 not found`
3. Updated repo and installed libc6
4. Re-ran install_makemkv.sh, which fails
5. Updated docker container `apt upgrade`, to see if packages were out of date, no change (still fails)

Error:
```bash
...
makemkv-oss-1.17.8/Makefile.in
+ rm makemkv-oss.tgz
+ cd makemkv-oss
+ '[' -f configure ']'
+ ./configure --prefix=/usr/local
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
checking how to print strings... printf
checking for gcc... gcc
checking whether the C compiler works... no
configure: error: in `/root/makemkv-oss':
configure: error: C compiler cannot create executables
See `config.log' for more details
```

Details in config.log (also attached)
```bash
...
gcc version 9.4.0 (Ubuntu 9.4.0-1ubuntu1~20.04.2) 
configure:3940: $? = 0
configure:3929: gcc -V >&5
gcc: error: unrecognized command line option '-V'
gcc: fatal error: no input files
compilation terminated.
configure:3940: $? = 1
configure:3929: gcc -qversion >&5
gcc: error: unrecognized command line option '-qversion'; did you mean '--version'?
gcc: fatal error: no input files
compilation terminated.
configure:3940: $? = 1
configure:3929: gcc -version >&5
gcc: error: unrecognized command line option '-version'
gcc: fatal error: no input files
compilation terminated.
configure:3940: $? = 1
configure:3960: checking whether the C compiler works
configure:3982: gcc    conftest.c  >&5
/usr/bin/ld: cannot find Scrt1.o: No such file or directory
/usr/bin/ld: cannot find crti.o: No such file or directory
collect2: error: ld returned 1 exit status
configure:3986: $? = 1
...
```

Potentially due to changes, when updating
```bash
# apt install libc6
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  binfmt-support dh-strip-nondeterminism dpkg-dev dwz gettext gettext-base gir1.2-freedesktop gir1.2-gdkpixbuf-2.0
  gir1.2-harfbuzz-0.0 gir1.2-rsvg-2.0 groff-base icu-devtools imagemagick-6-common intltool-debian lib32gcc-s1
  lib32stdc++6 libarchive-zip-perl libc-dev-bin libc6-i386 libcairo-script-interpreter2 libclang-common-10-dev
  libclang-cpp10 libclang1-10 libcroco3 libcrypt-dev libdebhelper-perl libdjvulibre-text libdjvulibre21 libexif12
  libffi-dev libfile-stripnondeterminism-perl libgc1c2 libglib2.0-dev-bin libgraphite2-dev libharfbuzz-gobject0
  libharfbuzz-icu0 libice-dev libilmbase-dev libilmbase24 libjbig-dev liblcms2-dev libllvm10 liblqr-1-0
  liblqr-1-0-dev libltdl-dev liblzo2-2 libmagick++-6-headers libmagick++-6.q16-8 libmagickcore-6-arch-config
  libmagickcore-6-headers libmagickcore-6.q16-6 libmagickcore-6.q16-6-extra libmagickwand-6-headers
  libmagickwand-6.q16-6 libobjc-9-dev libobjc4 libomp-10-dev libomp5-10 libopenexr-dev libopenexr24 libpcre16-3
  libpcre2-32-0 libpcre2-posix2 libpcre32-3 libpcrecpp0v5 libpfm4 libpipeline1 libpixman-1-dev libsepol1-dev
  libsm-dev libsub-override-perl libtiffxx5 libturbojpeg libuchardet0 libwmf0.2-7 libxcb-render0-dev
  libxcb-shm0-dev libxrender-dev libxt-dev libxt6 libz3-4 libz3-dev linux-libc-dev llvm-10 llvm-10-runtime
  llvm-10-tools man-db po-debconf python3-pygments python3-yaml
Use 'apt autoremove' to remove them.
The following additional packages will be installed:
  libblkid1 libc-bin libc-dev-bin libc6-i386 libexpat1 libfreetype6 libharfbuzz0b libncurses6 libncursesw6
  libtinfo6 locales zlib1g
Suggested packages:
  manpages glibc-doc
Recommended packages:
  manpages manpages-dev libc-devtools libnss-nis libnss-nisplus
The following packages will be REMOVED:
  build-essential clang clang-10 debhelper dh-autoreconf g++ g++-9 libass-dev libblkid-dev libbz2-dev libc6-dev
  libcairo2-dev libdjvulibre-dev libdvd-pkg libdvdcss-dev libdvdcss2 libexif-dev libexpat1-dev libfontconfig1-dev
  libfreetype-dev libfreetype6-dev libgdk-pixbuf2.0-dev libglib2.0-dev libharfbuzz-dev libicu-dev libjpeg-dev
  libjpeg-turbo8-dev libjpeg8-dev libmagick++-6.q16-dev libmagick++-dev libmagickcore-6.q16-dev
  libmagickwand-6.q16-dev libmount-dev libncurses-dev libnuma-dev libpcre2-dev libpcre3-dev libpng-dev
  libpython3-dev libpython3.8-dev librsvg2-dev libselinux1-dev libstdc++-9-dev libtiff-dev libtinfo-dev libtool
  libtool-bin libturbojpeg0-dev libwmf-dev libxml2-dev libzbar-dev llvm-10-dev python3-dev python3.8-dev uuid-dev
  zlib1g-dev
The following packages will be upgraded:
  libblkid1 libc-bin libc-dev-bin libc6 libc6-i386 libexpat1 libfreetype6 libharfbuzz0b libncurses6 libncursesw6
  libtinfo6 locales zlib1g
13 upgraded, 0 newly installed, 56 to remove and 177 not upgraded.
Need to get 12.4 MB of archives.
After this operation, 399 MB disk space will be freed.
Do you want to continue? [Y/n] 
```

[config.log](https://github.com/user-attachments/files/17307580/config.log)

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,6 +1,6 @@
 ###########################################################
 # base image, used for build stages and final images
-FROM phusion/baseimage:focal-1.2.0 AS base
+FROM phusion/baseimage:jammy-1.0.4 AS base
 RUN mkdir /opt/arm
 WORKDIR /opt/arm
 
@@ -36,10 +36,6 @@ RUN install_clean \
         vim \
         # arm extra requirements
         scons swig libzbar-dev libzbar0
-
-# add the PPAs we need, using add-ppa.sh since add-apt-repository is unavailable
-COPY ./scripts/add-ppa.sh /root/add-ppa.sh
-RUN bash /root/add-ppa.sh ppa:mc3man/focal6
 
 ###########################################################
 # install deps specific to the docker deployment

